cmd/cl-adm/cmd/create/create_peer.go

```diff
@@ -23,12 +23,15 @@ type PeerOptions struct {
 	Name string
 	// Dataplanes is the number of dataplanes to create.
 	Dataplanes uint16
+	// DataplaneType is the type of dataplane to create (envoy or go-based)
+	DataplaneType string
 }
 
 // AddFlags adds flags to fs and binds them to options.
 func (o *PeerOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&o.Name, "name", "", "Peer name.")
 	fs.Uint16Var(&o.Dataplanes, "dataplanes", 1, "Number of dataplanes.")
+	fs.StringVar(&o.DataplaneType, "dataplaneType", "envoy", "Type of dataplane")
```

Comment:
Few suggestions:
- Rename CLI arg from `dataplaneType` to `dataplane-type`.
- Add possible values to description (i.e. "Type of dataplane. Supported values: "envoy" (default), "go".
- Define the possible values as const strings. e.g:
```
	// DataplaneTypeEnvoy represents an envoy-type dataplane.
	DataplaneTypeEnvoy = "envoy"
	// DataplaneTypeGo represents a go-type dataplane.
	DataplaneTypeGo = "go"
```